src/main/java/az/edu/turing/modul1/Task2B.java

```diff
@@ -0,0 +1,68 @@
+package az.edu.turing.modul1;
+
+import java.util.Scanner;
+
+public class Task2B {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("---------Welcome to the Week's planner----------");
+        String[][] weeks = new String[7][2];
+        weeks[0][0] = "Sunday";
+        weeks[0][1] = "do home Work!";
+        weeks[1][0] = "Monday";
+        weeks[1][1] = "learning English ";
+        weeks[2][0] = "Tuesday";
+        weeks[2][1] = "watching film";
+        weeks[3][0] = "Wednesday";
+        weeks[3][1] = "work in DataStructures";
+        weeks[4][0] = "Thursday";
+        weeks[4][1] = "have a fun with friends";
+        weeks[5][0] = "Friday";
+        weeks[5][1] = "go to the fitness";
+        weeks[6][0] = "Saturday";
+        weeks[6][1] = "run in the morning!";
+
+        while (true) {
+            System.out.println("""
+                    Choose the Menu:
```

Comment:
the condition of the problem is not like that, we just receive an input and therefore we already choose an action

Reply:
I looked at the children's projects, most of them did not write the function of changing the task of a certain day, but in the condition of the task, there was also a change of the task of the day.

Reply:
I agree with you, but in condition don't say that, show user menu and then do action for that, in condition said that wait input from user (as a String) and do action based on this value